include/deal.II/grid/grid_tools.h

```diff
@@ -2379,7 +2380,8 @@ namespace GridTools
     const unsigned int                                            direction,
     const Tensor<1, FaceIterator::AccessorType::space_dimension> &offset =
       Tensor<1, FaceIterator::AccessorType::space_dimension>(),
-    const FullMatrix<double> &matrix = FullMatrix<double>());
+    const FullMatrix<double> &matrix    = FullMatrix<double>(),
+    const double              tolerance = 1e-10);
```

Comment:
I understand that 1e-10 was the previous value of the hardcoded tolerance. I have a quick question, is this value scaled with the "scale" or "length" of the mesh within the code? What I mean by this is that if this is an absolute tolerance or a relative tolerance. I think it would be better to name it specifically as to what it is. Given how it is used below, I think this is an absolute tolerance since it seems to have units of length.